src/code/z_skelanime.c

```diff
@@ -1024,10 +1028,11 @@ void AnimationContext_Update(PlayState* play, AnimationContext* animationCtx) {
         AnimationContext_LoadFrame, AnimationContext_CopyAll,   AnimationContext_Interp,
         AnimationContext_CopyTrue,  AnimationContext_CopyFalse, AnimationContext_MoveActor,
     };
-    AnimationEntry* entry;
+    AnimationEntry* entry = entry = animationCtx->entries;
 
```

Comment:
is this fake or does this make sense? lol

Reply:
Whoops copy paste error. Don't need the second entry.